CODEOWNERS

```diff
@@ -170,6 +175,9 @@
 /dts/bindings/*/st*                       @erwango
 /dts/bindings/sensor/ams*                 @alexanderwachter
 /dts/bindings/*/sifive*                   @mateusz-holenko @kgugala @pgielda @nategraff-sifive
+/dts/bindings/*/vexriscv,intc0.yaml       @mateusz-holenko @kgugala @pgielda
```

Comment:
Can you change this to be:
```
/dts/bindings/*/vexrisc*
/dts/bindings/*/litex*
```

there appears to be some bug if we have `,` in this file.

Reply:
@galak I refactored CODEOWNERS as suggested.